wrangler-service/src/main/java/co/cask/wrangler/service/gcs/GCSService.java

```diff
@@ -154,8 +153,9 @@ private boolean validateConnection(String connectionId, Connection connection,
   @GET
   @Path("/connections/{connection-id}/gcs/explore")
   public void exploreGCS(HttpServiceRequest request, HttpServiceResponder responder,
-                              @PathParam("connection-id") final String connectionId,
-                              @QueryParam("path") String path) {
+                         @PathParam("connection-id") final String connectionId,
+                         @QueryParam("path") String path,
+                         @QueryParam("bucket-limit") @DefaultValue("1000") int bucketLimit) {
```

Comment:
didn't notice before, but I think this should be named 'limit' and not 'bucket-limit' for both endpoints